code/game/objects/items.dm

```diff
@@ -630,3 +636,11 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 /obj/item/proc/microwave_act(obj/machinery/microwave/M)
 	if(M && M.dirty < 100)
 		M.dirty++
+
+/obj/item/MouseEntered(location,control,params)
+	if(held)
+		openToolTip(usr,src,params,title = name,content = "[desc]<br>[force ? "<b>Force:</b> [force]" : ""]",theme = "")

```

Comment:
We already have health analyzers that give exact breakdowns of how much damage each part of a person has taken.